apps/org/models.py

```diff
@@ -23,3 +30,27 @@ def save(self, **kwargs):
             self.set_unique_slug()
 
         super().save(**kwargs)
+
+
+class ResourceGrant(models.Model):
+    """A model to track which Organizations have access to which users' resources."""
+    organization = models.ForeignKey(
+        Organization,
+        on_delete=models.CASCADE
+    )
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+    resource_class = models.CharField(
+        max_length=255,
+        choices=RESOURCE_CHOICES,
+        default=RESOURCE_CHOICES[0]
+    )
+    provider_name = models.CharField(max_length=255)
```

Comment:
Move this to a `property` instead of a field. Grab the name from the stored `<resource_class>.name`